addons/web/static/tests/views/kanban_view_tests.js

```diff
@@ -1798,74 +1798,247 @@ QUnit.module("Views", (hooks) => {
+                    <templates>
+                        <t t-name="kanban-box">
+                            <div><field name="foo"/></div>
+                        </t>
+                    </templates>
+                </kanban>`,
+            groupBy: ["category_ids"],
+            async mockRPC(route, { method, args, kwargs }) {
+                assert.step(method || route);
+                if (method === "name_create") {
+                    assert.strictEqual(args[0], "new partner");
+                    const { default_category_ids } = kwargs.context;
+                    assert.deepEqual(default_category_ids, [6]);
+                }
+            },
+        });
+
+        assert.containsOnce(
+            target,
+            ".o_kanban_group:nth-child(2) .o_kanban_record",
+            "gold column should contain one records"
+        );
+
+        // click on 'Create', fill the quick create and validate
+        await quickCreateRecord(1);
+        await editQuickCreateInput("display_name", "new partner");
         await validateRecord();
 
-        assert.containsN(target, ".o_kanban_group:first-child .o_kanban_record", 3);
+        assert.containsN(
+            target,
+            ".o_kanban_group:nth-child(2) .o_kanban_record",
+            2,
+            "gold column should contain two records"
+        );
 
         assert.verifySteps([
             "get_views",
             "web_read_group", // initial read_group
             "web_search_read", // initial search_read (first column)
             "web_search_read", // initial search_read (second column)
-            "get_views", // form view in quick create
             "onchange", // quick create
-            "create", // should perform a create to create the record
+            "name_create", // should perform a name_create to create the record
             "read", // read the created record
             "onchange", // reopen the quick create automatically
         ]);
     });
 
+    QUnit.test("quick create record in grouped on m2m (field in the form view)", async (assert) => {
+        serverData.views["partner,some_view_ref,form"] = `
+            <form>
+                <field name="foo"/>
+                <field name="category_ids" widget="many2many_tags"/>
+            </form>`;
+
+        await makeView({
+            type: "kanban",
+            resModel: "partner",
+            serverData,
+            arch: `
+                <kanban on_create="quick_create" quick_create_view="some_view_ref">
+                    <templates>
+                        <t t-name="kanban-box">
+                            <div><field name="foo"/></div>
+                        </t>
+                    </templates>
+                </kanban>`,
+            groupBy: ["category_ids"],
+            async mockRPC(route, { method, args, kwargs }) {
+                assert.step(method || route);
+                if (method === "name_create") {
```

Comment:
there's no name_create in this case
you should do assertions on the create instead